grammar.js

```diff
@@ -549,6 +560,19 @@ module.exports = grammar({
         )
       ),
 
+    let_heredoc: ($) =>
+      seq(
+        '=<<',
+        repeat(alias($._let_heredoc_parameter, $.parameter)),
+        alias($._let_heredoc_marker, $.marker_definition),
+        optional($.comment),
+        '\n',
+        alias(repeat($._heredoc_line), $.body),
+        alias($._heredoc_end, $.endmarker),
+      ),
+
+    _let_heredoc_parameter: ($) => /[a-z]\S*/,
```

Comment:
Aren't these from a specific set ? If they do, I'd rather have:
```suggestion
    _let_heredoc_parameter: ($) => choice('trim', 'eval'),
```

And also, could you add a little comment so that we can easily find the reference of this in the documentation ?

Reply:
I initially thought it might be better for forward compatibility to define it the way I did given that these always start with a lowercase character, but you're right, specifying the list of valid `:let-heredoc` parameters would simplify things.

NB:`eval` only exists in Vim at the time I'm writing this.